controllers/groupsController.js

```diff
@@ -20,4 +22,64 @@ const list = async (req, res) => {
   }
 };
 
-module.exports = { list };
+const summary = async (req, res) => {
+  if (!isValidString(req.params.uid) || isEmptyString(req.params.uid)) {
+    return res.status(400).send({ message: "[InvalidUid] Error occured" });
+  }
+
+  try {
+    const postUpdateNewest = [];
+    const { uid } = req.params;
+    const keywordListResult = await keywordModel.find({ ownerUid: uid }).exec();
+    const keywordIdList = keywordListResult.map((keyword) => keyword._id);
+    const postCreatedLast = [];
+
+    for await (const id of keywordIdList) {
+      const postResult = await postModel.find({ keywordId: id }).sort({ createdAt: -1 }).exec();
+      postCreatedLast.push({
+        keywordId: postResult[0].keywordId,
+        createdAt: postResult[0].createdAt,
+      });
```

Comment:
```suggestion
    if (postResult.length !== 0) {
        postCreatedLast.push({
          keywordId: postResult[0].keywordId,
          createdAt: postResult[0].createdAt,
        });
      }
```
추가 테스트 진행해서 수정이 필요한 부분입니다. 쿼리 결과가 존재할 때에만 진행되도록 수정 예정입니다.
아래 70번째 라인도 동일하게 처리예정입니다. :)

Reply:
길이가 숫자이다 보니 `postResult.length !== 0` 보다는 `postResult.length > 0` 으로 조건을 설정하는 게 더 좋을 것 같은데, 진태님 의견은 어떠실까요?

Reply:
좋습니다! 의도가 더 적절하게 드러날 것 같습니다👍